src/components/project/ProjectBasicsPane.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import FormControl from "@material-ui/core/FormControl";
+import TextField from "@material-ui/core/TextField";
+import OrganizationSelect from "../../components/general/OrganizationSelect.js";
+
+export default function ProjectBasicsPane(props) {
+  return (
+    <>
+      <OrganizationSelect
+        organizations={props.organizations}
+        handleChange={props._handleOrganizationChange}
+        value={props.organization_id}
+      ></OrganizationSelect>
+      <FormControl fullWidth>
+        <TextField
+          id="project_name"
```

Comment:
HTML element IDs are global, and subtle weird things can happen when multiple elements have the same ID. Could we give these random IDs instead?

I could see defining a function called `randomIdForLabel` to make the intent clearer.

```jsx
function randomIdForInputLabel() {
  return `${Date.now().toString()}-${Math.random().toString()}`;
}

// ...

<TextField
  id={randomIdForInputLabel()}
  // ...
/>
```

The same comment applies for other `id`s in this PR.